integr8ly/integreatly-operator
# Cherry pick 2.9.2

# Description
 - [x] https://issues.redhat.com/browse/MGDAPI-2441 - commits 1-4
 - [x] https://issues.redhat.com/browse/MGDAPI-2278 - commits 6-9
 - [x] https://issues.redhat.com/browse/MGDAPI-2253 - commits 6-9
 - [x] https://issues.redhat.com/browse/INTLY-10768 - commits 6-9
 - [x] https://issues.redhat.com/browse/MGDAPI-2313 - commit 5
 - [x] https://issues.redhat.com/browse/INTLY-10773 - commit 15-16
 - [x] https://issues.redhat.com/browse/INTLY-10775 - commit 11-13

## Verification

- Install OSD 4.7.21
- Installed RHMI 2.9.1 using index -> quay.io/laurafitzgerald/integreatly-index:2.9.1
- Upgraded to RHMI 2.9.2 using index -> quay.io/laurafitzgerald/integreatly-index:2.9.2
      - CRO version is 0.30.0 
      - AMO version is 1.6.2 
      - Verify UPS is version 0.5.1  and alerts are using the correct queries as shown in this [pr](https://github.com/aerogear/unifiedpush-operator/pull/105)
      - Follow the verification steps from https://github.com/integr8ly/integreatly-operator/pull/2012 to verify that all alerting and dashboards are working as expected
- Upgrade to 4.8
     - Verify that the UPS alert isn't firing
     -  Verify 3Scale and OAuth routes don't have `router-default.` in them as described in https://github.com/integr8ly/integreatly-operator/pull/2003
     -  Follow the verification steps from https://github.com/integr8ly/integreatly-operator/pull/2012 to verify that all alerting and dashboards are working as expected





## Checklist
